dialect/sql/schema/mysql.go

```diff
@@ -377,12 +380,15 @@ func (d *MySQL) prepare(ctx context.Context, tx dialect.Tx, change *changes, tab
 }
 
 // scanColumn scans the column information from MySQL column description.
+// nolint:funlen
```

Comment:
`scanColumn` went over the 80 max with my additions, though I think it's hard an unnecessary to break this up

Reply:
Looks like @a8m has a change here to bump this limit: https://github.com/ent/ent/pull/1703/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9R11

Reply:
>  I think it's hard an unnecessary to break this up

Agree. Please change `.golangci.yml` as mentioned by @tarrencev